transport/src/main/java/io/netty/channel/ChannelInboundHandler.java

```diff
@@ -18,83 +18,9 @@
 /**
  * {@link ChannelHandler} which adds callbacks for state changes. This allows the user
  * to hook in to state changes easily.
+ *
+ * @deprecated use {@link ChannelHandler}
  */
+@Deprecated
```

Comment:
Given this is netty 5 and we are far from maintaining source compatibility, shouldn't these just be deleted rather than deprecated?

Reply:
I just did this for "smoother" migration. That said I think at some point we can go over all the `@deprecated` stuff and decide if we want to keep it or not.